charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -132,8 +133,16 @@ data:
                 {{ if $account.readPermissions }}--read-permissions {{ join " --read-permissions " $account.readPermissions }}{{ end }} \
                 {{ if $account.writePermissions }}--write-permissions {{ join " --write-permissions " $account.writePermissions }}{{ end }} \
                 --provider-version v2
+    EOF
     {{- end }}
 
+    # Run the commands in parallel over the account names limited by the specified parallelism factor.
+    xargs -P {{ .Values.halyard.installParallelism }} -I {} sh -c 'eval "$1"' - {} <<'EOF'
+    {{- range $name, $account := .Values.kubeConfig.accounts }}
```

Comment:
In other words, this returns essentially a list of:

```
source account_1
source account_2
...
source account_n
```

which is then executed via `eval` (in parallel).

Reply:
Nice improvement. Perhaps not for this PR, but as these inline YAML scripts become increasingly complex, should we eventually create them as actual separate `.sh` files (and only template them in yaml manifests)? E.g. smth like [this](https://github.com/PicnicSupermarket/picnic-infra-kubernetes/blob/71f9b6b06d2d2a0404443e92130024f4de82ee95/targets/global-prod/platform/nexus/nexus.values.yaml.gotmpl#L189).

Reply:
`.gotmpl` is only available when using Helmfile IIUC, so not sure if that's an option. We can split off parts to different files and template there though, but if we do that, I'd rather do that in a separate PR.